src/site/markdown/change-report-gh.md

```diff
@@ -44,6 +45,17 @@ Changelog {#changelog}
 | [0.1.0](#rel_1_0)   | 2003-07-08 |                      |
 | [0.0.1](#rel_0_1)   | 2003-05-31 |                      |
 
+## Release 1.2.0 - 2023-XX-01 {#rel_1_2_0}
+
+The following issues have been addressed:
+
+* The `locale` external character encoding option reverting to US-ASCII
+* Unnecessary string copying when sending a logging event to an appender
+* `Qt support/integration` not supporting a QString value in a LOG4CXX_XXXXX macro
+* A segfault during process exit when LogManager::shutdown is not called.
+
+Note: the `LOG4CXX_CHARSET` cmake option (external character encoding) default value has changed to `utf-8`
+
```

Comment:
should also add:
* hexdump utility added to log bytes
* Ability to remove HierarchyEventListener(#233)

For the Qt support, I would phrase it more that "LOG4CXX_XXXX macros now support logging of QStrings directly" to indicate that this is a new feature.